backend/lib/json_web_token.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class JsonWebToken
```

Comment:
Can you add simple unit test? That it does what it should be doing?

Reply:
Changes to `JwtService`

and here's the [test](https://github.com/ronsigter/bookshelf/pull/3/files#diff-6a6de5006563af3b1f494205688914a1576cb5cba4a2f938f7523357d26cf254R1)

Regarding the test here, It feels like I was testing the `JWT gem` method of `encode` and `decode` 😓